src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-document-card/index.js

```diff
@@ -105,7 +105,7 @@ export default {
                 return `sw-order-document-settings-${subComponentName}-modal`;
             }
 
-            return `sw-order-document-settings-${subComponentName}-modal`;
+            return `sw-order-document-settings-modal`;
```

Comment:
I don't know how this works and how the usage is intended. but shouldn't you provide a custom modal, if you have custom document type? :thinking:

Reply:
@mitelg No. This is nowhere documented and doesn't even make any sense.

Both statements return the same string. At first you are checking if the component is available and returning the custom modal. If it doesn't exist the custom modal will also be returned. Making more sense to return default modal. It doesn't break the custom documents and will work much better.

Reply:
ah yeah right, I see. I should have opened the lines above :sweat_smile: